languages/anaconda3/template/src/Algorithm.py

```diff
@@ -0,0 +1,36 @@
+import Algorithmia
+# API calls will begin at the apply() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+def apply(input):
+    return "hello {}".format(str(input))
+
+# Here is an example of an advanced form of an algorithm function,
+# which also uses a load function (see the example below)
+# -- ADVANCED ALGORITHM USAGE --
+# def apply(input, keras_model):
+#     prediction = keras_model.predict(input)
+#     result = {"class": prediction[0], "confidence": prediction[1]}
+#     return result
+
```

Comment:
Very nit and not necessarily suggesting you bother changing it at this point, just for future reference, documentation can/should go in docstrigs